src/components/ui/TextBoxSearch.scss

```diff
@@ -0,0 +1,25 @@
+.widget-text-box-search .search-bar {
+    position: relative;
+}
+
+.widget-text-box-search .search-bar input {
+    display: inline-block;
+    padding-left: 10px;
+}
+
+.widget-text-box-search .search-bar .btn-transparent {
+    border: none;
+}
+
+.widget-text-box-search .search-bar button {
+    position: absolute;
+    right: 2px;
+    top: 7px;
+}
+
+.widget-text-box-search .search-bar > span {
+    left: 10px;
+    position: absolute;
+    top: 9px;
+}
+
```

Comment:
Some refactoring to be done. Change the entire widget to be the scss way. Look at widgets like pagination, richtext and charts

Reply:
Refactoring done